src/oc/change/api/change.clj

```diff
@@ -45,9 +47,18 @@
 
   :delete! (fn [ctx] (let [user (:user ctx)
                            user-id (:user-id user)
-                           delete-item (read/delete! post-uuid user-id)]
+                           delete-item (read/delete! post-uuid user-id)
+                           container-id (-> ctx :request :body slurp)]
```

Comment:
@bago2k4 is sending the container-id in the body necessary if we already have the post-id? Can’t we lookup the container-id given the post-id?

Reply:
nope, CS has no simple way to determine the `container-id`. It has records, it could pull up a record from DynamoDB and check what's the `container-id` for it.
But there could be problems with that:
- post has been moved to a new container
- post was in draft board and then was published
probably others i can't think.
Also i prefer the client to send it, i made that optional, if there is a `container-id` then i send propagate the unread action to all the clients listening for the given container.